app/services/back_end_service.rb

```diff
@@ -1,29 +1,56 @@
 class BackEndService
   class << self
-    def base_url
-      'http://localhost:3000/api/v1'
-      # 'https://spotme-app-api.herokuapp.com/api/v1'
+    def get_user(google_id)
+      response = db_conn.get("/users/#{google_id}")
+      parse_json(response.body)
     end
 
-    def get_user(google_id)
-      receive_request("/users/#{google_id}")
+    def get_friends(user_id)
+      response = db_conn.get("/api/v1/users/#{user_id}/friendships")
+      parse_json(response.body)
+    end
+
+    def get_gyms(user_id)
+      response = db_conn.get("/api/v1/users/#{user_id}/gyms")
+      parse_json(response.body)
+    end
```

Comment:
Perhaps we update this to `get_gym_memberships` with endpoint `GET /api/v1/users/#{user_id}/gym_memberships`.  The `GymMembership` table now has a column for `gym_name` and `yelp_gym_id`.

Reply:
Agreed - Issue already opened to update Facade to call the new endpoint, which will take care of the update here as well.